### PR TITLE
[10.0][ADD] partner_socialmedia

### PR DESCRIPTION
**Pull request by funbaker**
This module adds social media information to res.partner, using char fields with url widgets.

### CODE REVIEW

File: partner_socialmedia/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Humanitarian Logistics Organisation e.V. - Stefan Becker
```
Comment (nikul-serpentcs) [inline partner_socialmedia/__init__.py]: @funbaker Remove Copyright from all `__init__`.py

File: partner_socialmedia/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Humanitarian Logistics Organisation e.V. - Stefan Becker
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Partner Socialmedia',
+    'summary': 'Add social media fields to contacts',
+    'version': '10.0.1.0.0',
+    'author': "humanilog",
+    'website': 'https://www.humanilog.org',
```
Comment (nikul-serpentcs) [inline partner_socialmedia/__manifest__.py]: @funbaker IMO Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

File: partner_socialmedia/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Humanitarian Logistics Organisation e.V. - Stefan Becker
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Partner Socialmedia',
+    'summary': 'Add social media fields to contacts',
+    'version': '10.0.1.0.0',
+    'author': "humanilog",
```
Comment (nikul-serpentcs) [inline partner_socialmedia/__manifest__.py]: @funbaker IMO author should like [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

File: partner_socialmedia/__init__.py (hunk 1 above)
Comment (funbaker) [reply partner_socialmedia/__init__.py]: Just wondering, there are many __init__.py's with copyright notices?

Comment (nikul-serpentcs) [reply partner_socialmedia/__init__.py]: @funbaker Only use `License`, Could you please check [here](https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py)

Comment (funbaker) [reply partner_socialmedia/__init__.py]: Done.

**Comment by funbaker** [comment]
Travis was failing because of another module.

**Review by hieulucky111 — APPROVED** [review]

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

**Comment by pedrobaeza** [comment]
Can you please squash commits all together and also to see Travis status?

**Comment by funbaker** [comment]
Squashed commits into one.

**Pull request merged**